jjb/paas/openshift-defaults.yml

```diff
@@ -0,0 +1,88 @@
+- defaults:
+    name: openshift-defaults
+    description: |
+        Maintainer: <a href="mailto:herlo@redhat.com>Clint Savage</a>
```

Comment:
Need to end the anchor with a quote. I think it's been like this since I added the bits. :/